components/x-topic-search/src/lib/get-suggestions.js

```diff
@@ -3,23 +3,43 @@ const addQueryParamToUrl = (name, value, url, append = true) => {
 	return append === true ? `${url}&${queryParam}` : `${url}?${queryParam}`;
 };
 
+const separateFollowedAndNot = (suggestions, followedTopics) => {
+	const matchingFollowedTopics = [];
+	const matchingNotFollowedTopics = [];
```

Comment:
`matchingUnfollowedTopics` might be more consistent with what's used elsewhere.

Reply:
@dan-searle I thought the word `unfollow` is an action to stop following something. For this case, the topics are not followed by the user. I thought `matchingNotFollowedTopics ` is more appropriate. However I may have used `unfollow` in the same case in somewhere :sweat_smile:

Reply:
I found the wrong use of `unfollowed` in only tests, However the wrong use will be removed by this PR.
https://github.com/Financial-Times/x-dash/pull/228

Reply:
Yeah, "unfollow" is the action, and "unfollowed" would describe that it's not currently followed.

Reply:
:open_mouth: In that case, I'm going to modify the name immediately :dash: